hudi-utilities/src/test/java/org/apache/hudi/utilities/testutils/UtilitiesTestBase.java

```diff
@@ -266,6 +268,17 @@ public void teardown() throws Exception {
     TestDataSource.resetDataGen();
   }
 
+  private static Map<String, String> sparkConf() {
+    Map<String, String> conf = new HashMap<>();
+    conf.put("spark.default.parallelism", "2");
+    conf.put("spark.sql.shuffle.partitions", "2");
+    conf.put("spark.executor.memory", "1G");
+    conf.put("spark.driver.memory", "1G");
+    conf.put("spark.hadoop.mapred.output.compress", "true");
+    conf.put("spark.ui.enable", "false");
```

Comment:
In a follow-up PR, we can unify these configs with `HoodieClientTestUtils#getSparkConfForTest` https://github.com/apache/hudi/blob/550367cdbda37fdf8a33d0a6ce71354915099c72/hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestUtils.java#L91

Reply:
Yes, I can try this along with some other test performance improvements